Makefile

```diff
@@ -67,6 +67,9 @@ README.md: youtube_dl/*.py youtube_dl/*/*.py
 CONTRIBUTING.md: README.md
 	$(PYTHON) devscripts/make_contributing.py README.md CONTRIBUTING.md
 
+issue_template: ISSUE_TEMPLATE.md youtube_dl/version.py
+	$(PYTHON) devscripts/make_issue_template.py ISSUE_TEMPLATE.md
```

Comment:
Whoops, so sorry, totally forgot about the makefile. Fixed now, ran `make` and it ran without errors.